cmd/root/root.go

```diff
@@ -43,10 +45,10 @@ const (
 	cfgGuestDNSServers      = "guest-dns-servers"
 	cfgGuestNTPServers      = "guest-ntp-servers"
 
-	cfgFlatcarChannel      = "flatcar-channel"
-	cfgFlatcarVersion      = "flatcar-version"
-	cfgFlatcarIgnition     = "flatcar-ignition"
-	cfgFlatcarIgnitionPath = "flatcar-ignition-dir"
+	cfgFlatcarChannel        = "flatcar-channel"
+	cfgFlatcarVersion        = "flatcar-version"
+	cfgFlatcarIgnitionString = "flatcar-ignition-string"
```

Comment:
I do prefer the old names of the flags/variables (especially for `cfgFlatcarIgnition`) but not a deal breaker.